crates/context/src/evm.rs

```diff
@@ -1,12 +1,16 @@
 use crate::setters::ContextSetters;
+use core::fmt::Debug;
 use core::ops::{Deref, DerefMut};
+use derive_where::derive_where;
 
+#[derive_where(Debug; CTX, INSP, I, P: Debug)]
```

Comment:
```suggestion
#[derive_where(Debug; CTX, INSP, I, P)]
```

Does it work without `: Debug`?

Reply:
No It gives error.

Reply:
what is the error?

Reply:
Same as below:
```
this can be handled by standard `#[derive(..)]`, use a `skip` or `incomparable` attribute, implement `Default` on an enum, or different generic type parametersrust-analyzer[macro-error](https://rust-analyzer.github.io/manual.html#macro-error)
```